api/db-cdr/generate-cdr/make-bq-criteria-tables.sh

```diff
@@ -0,0 +1,550 @@
+#!/bin/bash
```

Comment:
Are replacing `generate-cb-criteria-tables.sh` with this file? If so should we delete `generate-cb-criteria-tables.sh`?

Reply:
It will eventually replace it. It is a slow re-write. Once this new file is complete, we can drop the old one